src/components/IWD/2021/HowItWorks.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+	<section class="how-it-works section text-center">
+		<div class="row">
+			<p
+				v-html="howItWorksHeadline"
+				class="how-it-works__header large-text small-12 columns"
+			>
+			</p>
+			<p
+				v-html="howItWorksSubhead"
+				class="large-6 large-offset-3 columns"
+			>
+			</p>
+		</div>
+		<ol class="how-it-works__list row">
+			<li
+				v-for="step in stepsText"
+				:key="step.key"
+				class="how-it-works__li small-12 xxlarge-3 columns"
+			>
+				<img
+					v-if="step.image.url"
+					class="how-it-works__img"
+					:src="step.image.url"
+					:alt="step.image.description"
+				>
+				<p
+					v-html="step.headline"
+					class="how-it-works__subtitle"
+				>
+				</p>
+				<p v-html="step.subHeadline"></p>
+			</li>
+		</ol>
+	</section>
+</template>
+
+<script>
+// import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+
+export default {
+	props: {
+		content: {
+			type: Object,
+			default: () => {},
+		},
+	},
+	computed: {
+		howItworksText() {
+			return this.content?.contents?.find(({ key }) => key.indexOf('iwd-2021-homepage-how-it-works-text') > -1);
```

Comment:
I would recommend using 'homepage-how-it-works-text' as the string that you're checking for, so that this component can be used for other Contentful page layouts that have different prefixes

Reply:
good call